app/src/main/java/com/ifeins/tenbismonit/fragments/OverviewFragment.java

```diff
@@ -60,16 +65,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        mImageView = view.findViewById(R.id.overview_image);
-        mBudgetView = view.findViewById(R.id.budget_view);
-        mLunchesView = view.findViewById(R.id.lunches_view);
-        mTotalSpentView = view.findViewById(R.id.total_spent_view);
-        mAverageLunchView = view.findViewById(R.id.average_lunch_view);
-        mTodayBudgetView = view.findViewById(R.id.today_budget_view);
         mProgressBarView = view.findViewById(R.id.progress_bar);
         mProgressCaptionView = view.findViewById(R.id.progress_bar_caption);
         mLastUpdateView = view.findViewById(R.id.last_update_view);
         mErrorView = view.findViewById(R.id.error_view);
+        mRecyclerView = view.findViewById(R.id.overview_grid);
+
+        mRecyclerView.setHasFixedSize(true);
+        mLayoutManager = new GridLayoutManager(getContext(), 2);
+        mLayoutManager.setSpanSizeLookup(new GridLayoutManager.SpanSizeLookup() {
```

Comment:
What does this method does?

Reply:
Determines the size of the card. Since we decided that the first row card will span 2 columns, this method takes care of that.